src/sql/azdata.d.ts

```diff
@@ -3180,11 +3180,14 @@ declare module 'azdata' {
 		CSSStyles?: { [key: string]: string };
 	}
 
+	export type ThemedIconPath = { light: string | vscode.Uri; dark: string | vscode.Uri };
```

Comment:
you might want to give this a different name, vscode already have a class named ThemeIcon and we might add it as an option here in the future https://github.com/microsoft/azuredatastudio/blob/main/src/vs/vscode.d.ts#L800

Reply:
The name is different - and I think the difference between `ThemedIconPath` and `ThemeIcon` is enough to not be super confusing. 

If you have a better one though I'm open to suggestions. But it literally is a Themed icon path so I'm leaning towards keeping it as it is for clarity there.